### DISCUSSION TITLE
How to use v2rayA on Ubuntu

### DISCUSSION THREAD

**Discussion by erosman** [v2rayA/v2rayA#1277] [Q&A]
I have installed v2rayA on Ubuntu 23.10 via App center.
The icon shows on Apps and Firefox opens `http://localhost:2017/`.
I have added server/subscript (vless/tcp).

How do I start v2rayA?
I didn't find a Guide.

- How can it be set as a VPN?
- How can it be set as proxy to use in Firefox?

**Comment by vkapas** [comment]
I was looking at how to start using it too, and I figured it out eventually.

Just pick one of the servers with the Select button and connect to it with the Ready button on top.
![Untitled](https://github.com/v2rayA/v2rayA/assets/12103655/920721b6-abc8-43d6-aa04-36fa01fdbe63)
